valhalla/sif/edgelabel.h

```diff
@@ -28,7 +28,7 @@ class EdgeLabel {
    * TODO - without memset it warns of uninitialized data members
    */
   EdgeLabel() {
-    memset(this, 0, sizeof(EdgeLabel));
+    // memset(this, 0, sizeof(EdgeLabel));
```

Comment:
Is this comment intended?

Reply:
I _wasn't sure_. The comment four lines above says there are warnings without the memset. But I'm only seeing warnings _with_ the `memset`. I'd like to remove this line and the comment on line 28.

Reply:
Maybe @dnesbitt61 knows more?

Reply:
I do not remember what warnings I might have seen back in the earliest days of Valhalla to have made me add this, but the memset can be removed as well as the TODO in the comment.